decidim-core/app/helpers/decidim/translations_helper.rb

```diff
@@ -11,9 +11,17 @@ module TranslationsHelper
     #
     # Returns a String with the translation.
     def translated_attribute(attribute)
-      attribute.try(:[], I18n.locale.to_s).presence ||
-        attribute.try(:[], current_organization.default_locale).presence ||
+      return "" if attribute.nil?
+      return attribute unless attribute.is_a?(Hash)
+
+      attribute[I18n.locale.to_s].presence ||
+        attribute[current_organization.default_locale].presence ||
+        attribute[attribute.keys.first].presence ||
         ""
+      # attribute.try(:[], I18n.locale.to_s).presence ||
+      #   attribute.try(:[], current_organization.default_locale).presence ||
+      #   attribute.try(:[], attribute.keys.first).presence ||
+      #   ""
```

Comment:
Remove commented code

Reply:
Definitivelly it's a bad idea to code on sundays. I'll commited it by mistake. I will fix it.